code/game/objects/items/weapons/tools.dm

```diff
@@ -137,12 +153,56 @@
 	sharp = TRUE
 	edge = TRUE
 	drop_sound = 'sound/items/drop/knife.ogg'
+	var/random_color = TRUE
+	var/static/list/tool_colors = list(
+		"blue" = COLOR_BLUE,
+		"red" = COLOR_RED,
+		"pink" = COLOR_PINK,
+		"brown" = COLOR_BROWN,
+		"green" = COLOR_GREEN,
+		"cyan" = COLOR_CYAN,
+		"yellow" = COLOR_YELLOW
+	)
+
+/obj/item/wirecutters/Initialize()
+	. = ..()
+	if(random_color) //random colors!
+		var/our_color = pick(tool_colors)
+		add_atom_colour(tool_colors[our_color], FIXED_COLOUR_PRIORITY)
+		update_icon()
```

Comment:
I dont understand why you arent just picking from a list of the defines instead of a keyed list